### PR TITLE
Cleanup vendor package

### PR DESCRIPTION
**Pull request by liwenwu-amazon**
### Summary
This PR  is intended for  issue #27  and removes vendor dependency on
 * github.com/aws/amazon-ecs-cni-plugins/
 * github.com/aws/amazon-ecs-agent

### Test Done
```
make static
make lint
make vet
make unit-test
```

### CODE REVIEW

File: pkg/utils/logger/logger.go [hunk 1]
```diff
@@ -22,8 +22,8 @@ import (
 )
 
 const (
-	envLogLevel    = "ECS_CNI_LOGLEVEL"
-	envLogFilePath = "ECS_CNI_LOG_FILE"
+	envLogLevel    = "EKS_CNI_LOGLEVEL"
```
Comment (aaithal) [inline pkg/utils/logger/logger.go]: this should definitely not have `EKS` in the name. A suggestion here would be `AWS_VPC_CNI_K8S_LOGLEVEL`

Comment (liwenwu-amazon) [reply pkg/utils/logger/logger.go]: Done

**Comment by mattlandis** [comment]
@liwenwu-amazon I am OK merging this if you create an issue to clean up the tests that use ttime to not mock time.  It does the job of cleaning up the vendor directory, but leaves the technical debt it picked up from the dependencies it was pulling in.

**Comment by liwenwu-amazon** [comment]
#77  and move this change to master branch

**Pull request closed without merging**